sdk/spring/spring-cloud-azure-supported-spring.json

```diff
@@ -4,14 +4,30 @@
     "releaseStatus" : "SNAPSHOT",
     "snapshot" : true,
     "supportStatus" : "TODO",
+    "spring-boot-version" : "3.0.2-SNAPSHOT",
+    "spring-cloud-version" : "2022.0.0"
+  },
+  {
+    "current" : true,
+    "releaseStatus" : "GENERAL_AVAILABILITY",
+    "snapshot" : false,
+    "supportStatus" : "SUPPORTED",
+    "spring-boot-version" : "3.0.1",
+    "spring-cloud-version" : "2022.0.0"
+  },
+  {
+    "current" : false,
+    "releaseStatus" : "SNAPSHOT",
+    "snapshot" : true,
+    "supportStatus" : "END_OF_LIFE",
     "spring-boot-version" : "3.0.1-SNAPSHOT",
     "spring-cloud-version" : "2022.0.0-RC2"
   },
   {
     "current" : false,
     "releaseStatus" : "GENERAL_AVAILABILITY",
     "snapshot" : false,
-    "supportStatus" : "SUPPORTED",
+    "supportStatus" : "END_OF_LIFE",
     "spring-boot-version" : "3.0.0",
     "spring-cloud-version" : "2022.0.0"
```

Comment:
Should we sync other version mappings that have been released for Spring Cloud Azure 6.0?
such as, updating the releaseStatus to `GENERAL_AVAILABILITY` for RC2
![image](https://user-images.githubusercontent.com/19926623/209600850-b73fe78c-5901-41e3-aec6-f755471984b5.png)